app/models/booking.rb

```diff
@@ -4,30 +4,29 @@ class Booking < ApplicationRecord
   belongs_to :tour_detail
   belongs_to :user
   has_one :review, dependent: :destroy, class_name: Review.name
-  validates :phone, presence: true
+  validates :phone, presence: true, phone: true
   validates :date_start, presence: true
   validates :numbers_people, presence: true,
             numericality: {only_integer: true,
-                           greater_than: Settings.peoples_booking_tours_min}
+                           greater_than_or_equal_to:
+                           Settings.peoples_booking_min}
   validate :booking_date_in_future
+  validate :out_of_peoples
```

Comment:
```suggestion
  validate :out_of_peoples, if: -> numbers_people.present?
```

để khỏi check numbers_people.present? ở method out_of_peoples